### PR TITLE
Centers faction select button

### PR DESCRIPTION
**Pull request by alsoandanswer**
also fiddles with some text a little. I hate menu UI.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/client/preference_setup/occupation/occupation.dm [hunk 1]
```diff
@@ -447,15 +447,15 @@
 	dat += "</tr>"
 	dat += "</table><center><hr/>"
 
-	dat += "You can learn more about this faction on <a href='?src=\ref[user.client];JSlink=wiki;wiki_page=[replacetext(faction.name, " ", "_")]'>the wiki</a>.</center>"
+	dat += "You can learn more about this faction on <a href='?src=\ref[user.client];JSlink=wiki;wiki_page=[replacetext(faction.name, " ", "_")]'>the wiki</a>."
 
 	if (selected_faction == pref.faction)
-		dat += "<br>\[Faction Already Selected\]"
+		dat += "<br>\[Faction selected\]"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/occupation/occupation.dm:453]:
```suggestion
		dat += "<br>\[Faction Selected\]"
```
button should preferably remain title case.

File: code/modules/client/preference_setup/occupation/occupation.dm [hunk 2]
```diff
@@ -447,15 +447,15 @@
...
 	else if (faction.can_select(pref,user))
-		dat += "<br>\[<a href='?src=\ref[src];faction_select=[html_encode(selected_faction)]'>Select Faction</a>\]"
+		dat += "<br>\[<a href='?src=\ref[src];faction_select=[html_encode(selected_faction)]'>Select faction</a>\]"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/occupation/occupation.dm:455]:
```suggestion
		dat += "<br>\[<a href='?src=\ref[src];faction_select=[html_encode(selected_faction)]'>Select Faction</a>\]"
```
ditto.

File: code/modules/client/preference_setup/occupation/occupation.dm [hunk 3]
```diff
@@ -447,15 +447,15 @@
...
 	else
 		dat += "<br><span class='warning'>[faction.get_selection_error(pref, user)]</span>"
-
+	dat += "</center>"
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/occupation/occupation.dm:458]:
```suggestion

	dat += "</center>"
```

**Comment by alsoandanswer** [comment]
!bugfix

**Review by mikomyazaki — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**